scene/theme/theme_db.cpp

```diff
@@ -188,6 +189,49 @@ Ref<StyleBox> ThemeDB::get_fallback_stylebox() {
 	return fallback_stylebox;
 }
 
+// Theme item binding.
+
+void ThemeDB::bind_class_item(const StringName &p_class_name, const StringName &p_prop_name, const StringName &p_item_name, ThemeItemSetter p_setter) {
+	ERR_FAIL_COND_MSG(theme_item_binds[p_class_name].has(p_prop_name), vformat("Failed to bind theme item '%s' in class '%s': already bound", p_prop_name, p_class_name));
+
+	ThemeItemBind bind;
+	bind.class_name = p_class_name;
+	bind.item_name = p_item_name;
+	bind.setter = p_setter;
```

Comment:
Seems like everything other than the setter is unused.
Do you have any plans for these properties?

Reply:
Yes, these will be useful for generating docs and theme overrides. And just in general, it helps to debug and see which properties are actually bound here, because you can't easily peak into the setter.